src/capi/azext_capi/helpers/os.py

```diff
@@ -0,0 +1,19 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=missing-docstring
+
+import os
+
+
+def set_enviroment_variables(dic):
```

Comment:
These functions are simple and fairly obvious, but it's still a good practice to put docstrings everywhere.

Reply:
@mboersma, no please do hold me accountable! Definitely I am trying to follow best practices :)

Reply:
@mboersma, no please do hold me accountable! Definitely I am trying to follow best practices :)